Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -91,6 +102,29 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
         return options
     }
 
+    func isQuoted(_ message: _ChatMessage<ExtraData>) -> Bool {
```

Comment:
nit: we're checking if the message quotes another message, not if it's quoted by another message. We can either change the func name to `isQuotingAnotherMessage` or just keep the `if` untouched.

*the options name is `.quotedMessage` because it says whether the the cell should include a quoted message